src/Payments/PaymentCurrency.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Webit\WFirmaSDK\Invoices;
+
+final class PaymentCurrency
```

Comment:
Jak sądzę po dokumentacji modułu, ta klasa powinna raczej reprezentować kwotę wpłaty, a więc może zmieńmy ją na `PaymentAmount` oraz dodajmy w niej pole `amount`. Zatem mielibyśmy obiekt, który reprezentuje kwotę w dowolnej walucie.
```suggestion
final class PaymentCurrency
{
    public function __construct($value, $account, $valuePln = null)
    {
        $this->value = $value;
        $this->account = $account;
        $this->valuePln = $valuePln && $account === 'currency' ? $valuePln : null;
    }
}
```

Reply:
Dodajmy też konstruktor statyczny `fromPln($value)` oraz `fromCurrency($value, $valuePln = null)`. Trochę dziwne, że nie ma pola na przypisanie waluty jako takiej.

Reply:
Są trzy możliwości:
- faktura w PLN lub wyłączony moduł różnic kursowych (wtedy ani account, ani value_pln nie jest ustawiane - są nullami),
- faktura w walucie opłacona w walucie (wtedy value=wartość wpłaty w walucie, account=currency, value_pln=null),
- faktura w walucie opłacona w PLN (wtedy value=wartość wpłaty w walucie, account=pln, value_pln>0 - i powstają różnice kursowe).

Chciałem odseparować różnice kursowe (które są lub ich nie ma) od kwoty wpłaty (która jest zawsze i ma walutę faktury) - stąd osobna klasa tylko na to.